_component_design/context-drawer.md

```diff
@@ -0,0 +1,543 @@
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Sizes
+
+The context drawer features three available widths. Choose the best drawer size for your content.
+
+The drawer is responsive, and its width adjusts by increments of 2rem per breakpoint. At the smallest breakpoint, the drawer automatically fills 90% of the browser width, regardless of the drawer size.
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [caption:"Context drawer sizes"] [class:"image bg-light border"] %}
+<embed src="{{site.baseurl}}/assets/images/components/content-areas/context-drawer/context-drawer-sizes.png" width="637"/>
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+### Animation
+
+Use these animation guidelines when you construct a context drawer.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+#### Easing
+
+The side drawer remains off-screen until triggered by an action initiated by the user from the main content area.
+
+When triggered, the drawer immediately starts loading its contents and animates into view.
+
+When the drawer slides into view, it uses ease-out animation, which means that the drawer comes in fast and slows down as it reaches its stopping point.
+
+When the drawer slides out of view, it uses ease-in animation, which means that the drawer starts moving slowly and picks up speed as it moves off screen.
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [caption:"Context drawer easing animation"] [class:"image bg-light border"] %}
+<embed src="{{site.baseurl}}/assets/images/components/content-areas/context-drawer/context-drawer-animation-easing.png" width="675"/>
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+#### Duration
+
+Duration is the amount of time the drawer takes to animate from off-stage to on-stage, and from on-stage to off-stage.
+
+Each drawer size has a unique animation duration so that the visual weight and movement looks natural.
+
+The drawer slides out twice as fast as it slides in because the closing animation requires less user focus than does the user’s next task.
+
+**Animating into view**
```

Comment:
Any reason for not using heading 5 (so people can link directly to these ones)?

Reply:
Done.